### ISSUE TITLE
build symmetric adjacency matrix

### ISSUE THREAD

**Issue by amitiwari** [tkipf/kegra#36]
Hi, 

Just curious about this line in utils/load_data
adj = adj + adj.T.multiply(adj.T > adj) - adj.multiply(adj.T > adj)

Seems like it fails due to the comparison which makes it boolean. Is it even needed ? As I see it , the adjacency matrix is already symmetric with a shape of (2708, 2708).

**Comment by tkipf** [comment]
Yes, the adjacency matrix is "symmetric" in shape, but it is not necessarily a symmetric matrix (i.e., `A^T = A`). Maybe try a different python/numpy version or add an explicit conversion to float if you get an error. Feel free to submit a pull request with a fix.
